YesterdayApi.UserIdentity.Data/Core/Users/AspIdentityUserService.cs

```diff
@@ -0,0 +1,6 @@
+namespace YesterdayApi.UserIdentity.Data.Core.Users
+{
+    class AspIdentityUserService
```

Comment:
I don't get this one. Can you explain?

Reply:
Forgot to write the code xd, updated it.